html/semantics/forms/the-input-element/range.html

```diff
@@ -41,7 +41,7 @@ <h1>Input Range</h1>
       <input type="range" id="stepdown_beyond_min" min=3 max=11 value=6 step=3 />
       <input type="range" id="illegal_min_and_max" min="ab" max="f" />
       <input type="range" id="illegal_value_and_step" min=0 max=5 value="ppp" step="xyz" />
-      <input type="range" id="should_skip_whitespace" value=" 123"/>
+      <input type="range" id="should_skip_whitespace" value=" 50"/>
```

Comment:
What if we use another value than 50, for example " 49"?

The reason I am concerned about using 50 is that all browsers were getting 50 already when the value was " 123". Therefore, it is not surprising if they still get 50 when you update the value to " 50".

Since this test is about the leading space getting skipped, we need to make sure the browsers are actually parsing this properly and update the value. Therefore, we should test something else than 50, no?

Reply:
Done, thanks for the suggestion!